### ISSUE TITLE
Provide the ability to select kubernetes namespaces

### ISSUE THREAD

**Issue by slemeur** [containers/podman-desktop#8348]
### Is your enhancement related to a problem? Please describe

As a developer who is using Podman Desktop and the kubernetes features, I'm usually working with multiple namespaces. Unfortunately, with Podman Desktop, I can't select the namespace I want to visualize.

### Describe the solution you'd like

It would be nice to allow users to select the namespace to visualize the objects deployed within their Kubernetes environment.

One solution would be to allow a dropdown here:
<img width="958" alt="Screenshot 2024-08-05 at 15 07 07" src="https://github.com/user-attachments/assets/d4b72853-f496-4ea5-adca-ecc04d8f92df">

Also, it would be important to display which namespace is currently selected in the kubernetes selector + in the screen allowing to interact with kubernetes environments.

### Additional context

Linking to: https://github.com/containers/podman-desktop/issues/7191

**Comment by afbjorklund** [comment]
You can change it from the command-line, I guess that was "good enough" for the feature

* https://github.com/containers/podman-desktop/issues/4990

**Comment by shipsing** [comment]
![image](https://github.com/user-attachments/assets/6e48f4d3-10c3-4642-814d-5b95488d354a)

We have this field `Kubernetes Namespace` in the `Deploy to Kubernetes` page. I changed the namespace from `default` to  `kube-public`. I think user has the ability to change namespaces from the UI. We have mentioned the same in  [our doc](https://podman-desktop.io/docs/kubernetes/deploying-a-pod-to-kubernetes) on step no-4.
Also, when i change the kube context from `kind-cluster` to `dev-sandbox`, i do not see this `Kubernetes Namespace` field. So, seems like some sort of discrepancy.

Are we now going to provide this edit namespace ability from the Settings > Kubernetes section as well??

**Comment by deboer-tim** [comment]
@shipsing this screen is only changing the target for one deployment, and if you pick a namespace that you don't have a context for there's no easy way to view it from within Podman Desktop today.

Yes, the request above is specifically suggesting to allow editing it in the Kubernetes section.

**Comment by Bjego** [comment]
Yes this would be super helpfull - have a look into the vscode kubernetes plugin there you can select the namespace in the statusbar next to the cluster - maybe this could be a valid approach here as well

**Comment by evanshortiss** [comment]
Would love to see this feature happen. I ran this issue with viewing my Pods in Kind this week.

**Comment by Firewall** [comment]
All sub-issues closed. Coming with 1.18! 🚀
